apps/meteor/client/views/marketplace/helpers/sortAppsByMostRecentRequested.ts

```diff
@@ -0,0 +1,4 @@
+export const sortAppsByMostOrLeastRecentRequested = (
+	firstAppUnseenRequests: number | undefined,
+	secondAppUnseenRequests: number | undefined,
+) => (secondAppUnseenRequests || 0) - (firstAppUnseenRequests || 0);
```

Comment:
almost useless are we planning to create some .spec around this as well?

Reply:
I still stand by the idea that using named helper functions for complex callbacks improves readability for the code. Not only that but yes, we have a full test coverage of the marketplace planned for the next sprints...